packages/medusa-plugin-typesense/src/utils/transformer.ts

```diff
@@ -0,0 +1,43 @@
+import { variantKeys } from "@medusajs/types"
+
+const prefix = `variant`
+
+export const transformProduct = (product: any) => {
```

Comment:
should we type product with the `product` type from `@medusajs/types` instead of using `any`.